core/src/main/java/org/elasticsearch/action/admin/indices/analyze/TransportAnalyzeAction.java

```diff
@@ -462,96 +466,106 @@ private void analyze(TokenStream stream, Analyzer analyzer, String field, Set<St
         return extendedAttributes;
     }
 
-    private static CharFilterFactory[] getCharFilterFactories(AnalyzeRequest request, IndexSettings indexSettings, AnalysisRegistry analysisRegistry,
-                                                              Environment environment, CharFilterFactory[] charFilterFactories) throws IOException {
-        if (request.charFilters() != null && request.charFilters().size() > 0) {
```

Comment:
I've switched this around to return early rather than indent a bunch so the change look bigger than it is.